Makefile

```diff
@@ -1,17 +1,15 @@
+all: requirements quality test
+
 clean:
 	coverage erase
 	find . -name '*.pyc' -delete
 
 quality:
-	pep8 --config=.pep8 organizations
-	pylint --rcfile=pylintrc organizations
+	tox -e quality
 
 requirements:
 	pip install -e .    # Install this package and its dependencies
 	pip install -r test-requirements.txt
 
 test:
-	django-admin.py test organizations --settings=test_settings --with-coverage --cover-package=organizations
-	coverage report
-
-.PHONY: clean, quality, requirements, test
+	tox
```

Comment:
Relevant fact: you can run `tox -e <env>` to run tests in a single environment, such as `tox -e django18` or `tox -e quality`.